partner_stock_risk/tests/test_partner_stock_risk.py

```diff
@@ -0,0 +1,35 @@
+# -*- coding: utf-8 -*-
+# © 2016 Carlos Dauden <carlos.dauden@tecnativa.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from openerp.tests.common import TransactionCase
+
+
+class TestPartnerStocklRisk(TransactionCase):
+    def setUp(self):
+        super(TestPartnerStocklRisk, self).setUp()
+        self.partner = self.env['res.partner'].create({
+            'name': 'Partner test',
+            'customer': True,
+        })
+        self.product = self.env.ref('product.product_product_36')
+        self.quant = self.env['stock.quant'].create({
+            'qty': 100,
+            'location_id': self.env.ref('stock.stock_location_stock').id,
+            'product_id': self.product.id,
+        })
+        self.picking = self.env['stock.picking'].create({
+            'picking_type_id': self.env.ref('stock.picking_type_out').id,
```

Comment:
Kudos for adding tests, but [they fail](https://travis-ci.org/OCA/partner-contact/jobs/178355942#L1177-L1214).